File: src/main/java/com/theendercore/hydra/util/HydraCommands.java
package com.theendercore.hydra.util;

import com.mojang.brigadier.context.CommandContext;
import com.mojang.brigadier.exceptions.CommandSyntaxException;
import net.fabricmc.fabric.api.client.command.v2.ClientCommandManager;
import net.fabricmc.fabric.api.client.command.v2.FabricClientCommandSource;
import net.minecraft.server.command.ServerCommandSource;

import static com.theendercore.hydra.HydraMod.LOGGER;
import static com.theendercore.hydra.twitch.TwitchBot.*;

public class HydraCommands {
    public static int enable(CommandContext<FabricClientCommandSource> context) throws CommandSyntaxException {
//        return Enable(context.getSource());
        LOGGER.info(context.toString());
        return 1;
    }

    public static int disable(CommandContext<FabricClientCommandSource> context) throws CommandSyntaxException {
        return Disable();
    }

    public static int test(CommandContext<FabricClientCommandSource> context) throws CommandSyntaxException {

        return 1;
    }
}

File: src/main/kotlin/com/theendercore/hydra/util/HydraCommands.kt
package com.theendercore.hydra.util

import com.theendercore.hydra.twitch.TwitchBot

@Suppress("unused")
object HydraCommands {
    fun enable(): Int {
        var x = 0
        Thread {
            x = TwitchBot.enable()
        }.start()
        return x
    }

    fun disable(): Int {
        return TwitchBot.disable()
    }

    fun test(): Int {
        return 1
    }
}